src/config_file.cc

```diff
@@ -188,7 +188,17 @@ void CONFIG_TREC::record_tag( string config_tag, string config_value ) {
 	found = 1;
 	}
   }
-  
+
+  if (config_tag == "lmparam_delta" ) {
+	lmparam_delta = strtod(config_value.c_str(), NULL);
```

Comment:
The way Dir+ is implemented in the existing language model I was thinking it would be easier to comprehend the parameter names if they are kept consistent with the existing naming patterns. Going by how delta is named in Dir+ [PR](https://github.com/xapian/xapian/pull/111/files#diff-8a2d77838da5b7da44ddb21e3d97acb1R1167), I'll update this as `dirplus_delta`. Thanks for pointing this out :)